src/CommonITILValidation.php

```diff
@@ -1822,6 +1822,7 @@ public static function dropdownValidator(array $options = [])
             'rand'                => $params['rand'],
             'width'               => $params['width'],
             'required'            => $params['required'],
+            'aria_label'          => __('Validator type'),
```

Comment:
We are using Approver/Approval/Approve not Validator/Validation/Validate in the UI.